collections/Lat_th/MS_Lat_th_e_22.xml

```diff
@@ -51,132 +46,176 @@
                </msIdentifier>
                <head>Sermon collection; German, 1392</head>
                <msContents>
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem>
+                     <locus>(upper endleaf)</locus>
+                     <note>List of contents</note>
+                  </msItem>
                   <msItem xml:id="MS_Lat_th_e_22-item1" n="1">
+                     <locus>(fols. 1r–122v)</locus>
                      <title key="work_14583">Sermon collection</title>
-                     <rubric>Sermones quadragesimales <note>(front cover)</note>
-                     </rubric>
+                     <rubric>Sermones quadragesimales</rubric>
+                     <note>(front cover)</note>
+                     <incipit><hi rend="underline"><quote>Tu autem cum ieiunas unge capud tuum</quote> Mt vi.</hi> In hoc ieiunii capite necessarium nobis est ut taliter ieiunemus</incipit>
```

Comment:
the quote element puts this in italic by default, so I don't think the underline is necessary

Reply:
Using this because it also includes the citation in the MS – unnecessary detail?

Reply:
I think so. You can use `<cit> <quote> ... </quote> <bibl>Mt. vi</bibl></cit>` if you want though!